style.css

```diff
@@ -101,3 +120,139 @@ body {
   display: flex;
   justify-content: space-between;
 }
+
+/* Work section */
+.Work {
+  display: flex;
+  flex-direction: column;
+  flex-wrap: wrap;
+}
```

Comment:
- [ ] Following the project requirements, The main container of the projects should have `display:grid` instead `display: flex;`. Kindly work on that.